makeflow/src/makeflow.c

```diff
@@ -754,6 +761,11 @@ static int makeflow_check(struct dag *d)
 			if(skip_file_check || batch_fs_stat(remote_queue, f->filename, &buf) >= 0) {
 				continue;
 			}
+
+			if(f->source) {
```

Comment:
The makeflow is incorrectly reported as inconsistent when you clean twice without intermediate runs, this is because we don't load the mountfile when we are cleaning and rely on the makeflow log to provide source.